src/SprykerEco/Zed/Payone/Persistence/PayoneEntityManager.php

```diff
@@ -65,4 +67,33 @@ public function updatePaymentPayoneOrderItem(PaymentPayoneOrderItemTransfer $pay
 
         return $paymentPayoneOrderItemTransfer;
     }
+
+    /**
+     * @param \Generated\Shared\Transfer\PaymentDetailTransfer $paymentDataTransfer
+     * @param int $idOrder
+     *
+     * @return void
+     */
+    public function updatePaymentDetail(PaymentDetailTransfer $paymentDataTransfer, int $idOrder): void
+    {
+        $paymentEntity = $this->createPaymentByOrderId($idOrder)->findOne();
+        $paymentDetailEntity = $paymentEntity->getSpyPaymentPayoneDetail();
+
+        $paymentDetailEntity->fromArray($paymentDataTransfer->toArray());
+
+        $paymentDetailEntity->save();
+    }
+
+    /**
+     * @param int $idOrder
+     *
+     * @return \Orm\Zed\Payone\Persistence\SpyPaymentPayoneQuery
+     */
+    protected function createPaymentByOrderId(int $idOrder): SpyPaymentPayoneQuery
```

Comment:
* Do not see a reason to have this method.

Reply:
++